docs/_docs/ruby-101.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Ruby 101
+permalink: /docs/ruby-101/
+---
+
+Jekyll is written in Ruby so you need to understand some of the jargon to reach its full potential.
```

Comment:
hm, i feel like `understand some of its jargon` is a bit vague... can we maybe say "write a custom plugin" or "use a gemfile"?